internal/cli/task_cancel.go

```diff
@@ -26,13 +26,15 @@ func (c *TaskCancelCommand) Run(args []string) int {
 	}
 	ctx := c.Ctx
 
-	if len(c.args) == 0 && c.flagRunJobId == "" {
-		c.ui.Output("Task ID or Run Job Id required.\n\n%s", c.Help(), terminal.WithErrorStyle())
-		return 1
+	var taskId string
+	if len(c.args) > 0 {
+		taskId = c.args[0]
 	}
-	taskId := c.args[0]
 
-	if c.flagRunJobId != "" && taskId != "" {
+	if taskId == "" && c.flagRunJobId == "" {
+		c.ui.Output("Task ID or Run Job Id required.\n\n%s", c.Help(), terminal.WithErrorStyle())
```

Comment:
```suggestion
		c.ui.Output("Task Id or Run Job Id required.\n\n%s", c.Help(), terminal.WithErrorStyle())
```

Super small, but `ID` is Id throughout, so I thought it would make sense to lower the second char in this case too.